Add product lifecycle page and regions page (#2174)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-versions/product-lifecycles.md

---
title: "dbt Product lifecycles"
id: "product-lifecycles"
description: "Learn about dbt Labs' product lifecycles."
---

dbt Labs is directly involved with the maintenance of two products: 

- dbt Core: The [open-source](https://github.com/dbt-labs/dbt-core) software that’s freely available 
- dbt Cloud: The managed [SaaS solution](https://www.getdbt.com/signup) built on top of dbt Core

Any dbt feature will fall into one of the following lifecycle states:

### dbt Cloud

- **Beta:** Beta features may be made available for the purpose of customer testing and evaluation. These may not be feature-complete or fully stable. There may still be some planned additions and modifications to product behaviors while in Beta. Breaking changes may occur – although we will do our best to communicate them in advance, we may not always be able to do so. Beta features may not be fully documented, technical support may be limited, and service level objectives (SLOs) may not be provided.

> - **Preview (Private or Public):**  Preview features are stable and can be considered for production deployments. There may still be some planned additions and modifications to product behaviors before moving to General Availability. We may also introduce new functionality to Preview features that are not backward compatible. Preview features include documentation, technical support, and include service level objectives (SLOs). Features in Preview are generally provided at no extra cost, although they may become paid features in their Generally Available state.

Reviewer: Should say: "We may also introduce new functionality to Preview features that **is** not backward compatible"

Author: ah ok, so i changed it to **are** because "features" is plural but i think both are fine so will add your sugg

> - **Preview (Private or Public):**  Preview features are stable and can be considered for production deployments. There may still be some planned additions and modifications to product behaviors before moving to General Availability. We may also introduce new functionality to Preview features that are not backward compatible. Preview features include documentation, technical support, and include service level objectives (SLOs). Features in Preview are generally provided at no extra cost, although they may become paid features in their Generally Available state.

Author: ```suggestion
- **Preview (Private or Public):**  Preview features are stable and can be considered for production deployments. There may still be some planned additions and modifications to product behaviors before moving to General Availability. We may also introduce new functionality to Preview features that is not backward compatible. Preview features include documentation, technical support, and include service level objectives (SLOs). Features in Preview are generally provided at no extra cost, although they may become paid features in their Generally Available state.
```

> - **Preview (Private or Public):**  Preview features are stable and can be considered for production deployments. There may still be some planned additions and modifications to product behaviors before moving to General Availability. We may also introduce new functionality to Preview features that are not backward compatible. Preview features include documentation, technical support, and include service level objectives (SLOs). Features in Preview are generally provided at no extra cost, although they may become paid features in their Generally Available state.

Author: ```suggestion
- **Preview (Private or Public):**  Preview features are stable and can be considered for production deployments. There may still be some planned additions and modifications to product behaviors before moving to General Availability. We may also introduce new functionality to Preview features that is not backward compatible. Preview features include documentation, technical support, and include service level objectives (SLOs). Features in Preview are generally provided at no extra cost, although they may become paid features in their Generally Available state.
```

> - **Generally Available(GA):** Generally Available features are stable and can be considered for production deployments. GA features include documentation, technical support, and service level agreements (SLAs) apply.

Reviewer: Not a huge deal by any means, but I did prefer the phrasing in the original doc:

> Features are fully documented, technical support is provided, and service level agreements (SLAs) apply.

Author: no worries @azzam34 , just switched it to active voice because it's more beneficial for translation (in the future) and non English speakers. @runleonarun is also reviewing this too - Leona if you can also check this too please? 🙏

Reviewer 2: Here's my suggestion!
```suggestion
- **Generally Available(GA):** Generally Available features provide stable features that can be considered for production deployments. Service level agreements (SLAs) apply to GA features, and these features include documentation and technical support.
```
I have a question, @azzam34 and @mirnawong1  should we be promising support for new features? I thought support depended on your plan? Also I would refrain from saying "fully documented" just in case that means more than what we can provide with such a small team.

Author: I'll defer to @barberscott for the support question. my understanding is that for GA releases, the user can reach out to support, which may or may not lead to bug reports from the support team after they investigate.

- **Deprecated:** Features in this state are not actively worked on or enhanced by dbt Labs and will continue to function as-is until their removal date. 

- **Removed:** Removed features no longer have any level of product functionality or platform support.

### dbt Core

> 

Reviewer 2: We might want to point to the versioning page here. 
```suggestion

We release dbt Core in the following lifecycle states. Core releases follow semantic layer versioning, which you  can read more about in [About Core versions](/docs/dbt-versions/core).
```

> - **Unreleased:** We include this functionality in the next minor version prerelease. However, we make no commitments about its behavior or implementation. As maintainers, we reserve the right to change any part of it, or remove it entirely (with an accompanying explanation.)

Reviewer: We *will

> - **Unreleased:** We include this functionality in the next minor version prerelease. However, we make no commitments about its behavior or implementation. As maintainers, we reserve the right to change any part of it, or remove it entirely (with an accompanying explanation.)

Author: ```suggestion
- **Unreleased:** We will include this functionality in the next minor version prerelease. However, we make no commitments about its behavior or implementation. As maintainers, we reserve the right to change any part of it, or remove it entirely (with an accompanying explanation.)
```
